airbase/pom.xml

```diff
@@ -1391,6 +1389,33 @@
             </build>
         </profile>
 
+        <profile>
+            <id>surefire-use-testng</id>
+            <activation>
+                <property>
+                    <name>air.test.use-testng</name>
+                    <value>true</value>
+                </property>
+            </activation>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>org.apache.maven.plugins</groupId>
+                            <artifactId>maven-surefire-plugin</artifactId>
+                            <dependencies>
```

Comment:
Does this override the configuration above to use JUnit? If so, this will break projects that use both frameworks.

Reply:
Any project that uses both will need to include both explicitly. Surefire will detect JUnit or TestNg, but not both, unfortunately. The problem with including both in the base POM is that any project without TestNG on the classpath will fail. Ideally, the TestNG provider would be skipped if TestNG is not present, but instead it fails with `ClassNotFoundException`. There's no good way to conditionally add the dependency, since you can't active a profile with a Maven property, and there is no way to exclude it in a child project, since the dependencies section doesn't use `combine.self="override"` like configuration does.

Reply:
The downside is that if you have both, but don't include the providers, the TestNG tests will be silently skipped. But that seems better than forcing every module that doesn't use TestNG to include TestNG on the test classpath.

Reply:
I fixed the commits and added a description